### ISSUE TITLE
Separate Test sets for classification and Possible cleaning

### ISSUE THREAD

**Issue by rhn19** [goru001/inltk#49]
All the Classification datasets provided lack a separate test set. The reported accuracy is based on the validation set which was used to choose the best baseline model. This may cause issues in model generalization. Hence, there is a need for a separate test set to properly compare accuracy.
Another issue worth checking - The datasets have a few out-of-language characters (for example, Marathi Headlines have a few English characters and Arabic Numerals). These can be transliterated to the language specific characters and numerals. The datasets have ~50-100 duplicates which can be cleaned.

Following the above steps, I have created new splits for the Marathi Headlines Dataset ([link here](https://www.kaggle.com/rhn1903/marathi-news-dataset-cleaned)). I report the baseline accuracy on the test set using ALBERT-base pre-trained on the Marathi corpus [here](https://github.com/rhn19/indic-LM) (Acc:0.94, Kappa:0.89).

**Comment by goru001** [comment]
@rhn19 Thanks for reaching out. You're right about the train, val, test split. This was on my radar for a long time. I've changed the datasets against which models were benchmarked. You can check the updated table in README. Regarding cleaning of the datasets, a lot of work in this regard was done over [here](https://github.com/ai4bharat-indicnlp/indicnlp_corpus). So, instead of duplicating the work, I ended up using the same datasets for benchmarking. 

I'm glad to see that you benchmarked results using ALBERT. Feel free to raise PR into [NLP for Marathi](https://github.com/goru001/nlp-for-marathi) repo, if you want your results to be reported over there, or if you want to integrate transformers (ALBERT in this case), into iNLTK, go ahead - I'll be happy to help out if needed. I've wanted to integrate transformers for a long time now but have been running low with bandwidth.

Closing this ticket for now, as benchmarking datasets have now been changed. Feel free to reopen if you think I'm missing out on something! Thanks!
